docs/decisions/0018-units-as-containers.rst

```diff
@@ -0,0 +1,37 @@
+18. Modeling Units as a Concrete Implementation of the Container Capability
+=======================================================================
+
+Context
+-------
+
+The container capability is a generalized capability to hold different types of content. This decision focuses on modeling units as a concrete implementation of the container capability.
+
+Decisions
+---------
+
+All decisions from `0017-generalized-containers.rst`_ are still valid so that this decisions will build on top of them.
+
+.. _`0017-generalized-containers.rst`: 0017-generalized-containers.rst
+
+1. Units as Containers
+=======================
+
+- A unit is a concrete type of container that holds components.
+- A unit is a container, making it also a publishable entity.
+- A unit application will offer shared definitions for use by other unit subtypes.
```

Comment:
What is a "unit application"?

Reply:
> * Generalized containers (containers app–lowest level of these)
> * Selectors for dynamically selecting 0-N PublishableEntities, i.e. how we're going to do things like SplitTest and Randomized (selectors app, builds on containers).
> * Units (units app, builds on containers and selectors)–basically empty shells at the moment.

Can you include this whole example? That really makes it more clear.

Reply:
Yes, sure!